scripts/login.js

```diff
@@ -154,26 +154,26 @@ function checkDefaultDoWhop(person) {
 
 // For checking the pre-specified routing location. New.
 function retrieveUrl(location) {
-  if (location.match(/#(.+)/)[1]) {
+  if (location.match(/#(.+)/) != null && location.match(/#(.+)/)[1] != null) {
     var y = location.match(/#(.+)/)[1];
-    console.log('current location is', y);
     return y;
   } else {
     return null;
```

Comment:
🐞 When I sign in for the first time. retrieveUrl returns null. Then, setLandingTab is invoked with input set to null resulting in a TypeErrror: Cannot read property 'match' of null.